src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
```

Comment:
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-use-verbs-for-classinterface-names

how about `Drawable`?